graph/pull_v2.go

```diff
@@ -237,7 +237,7 @@ func (p *v2Puller) pullV2Tag(tag, taggedName string) (verified bool, err error)
 
 	out.Write(p.sf.FormatStatus(tag, "Pulling from %s", p.repo.Name()))
 
-	downloads := make([]downloadInfo, len(manifest.FSLayers))
+	var downloads []*downloadInfo
```

Comment:
probably wouldn't hurt to still use `make` but allocate it with size `0` and capacity for `len(manifest.FSLayers)`?